### ISSUE TITLE
[Support] Tests hang when using fixture

### ISSUE THREAD

**Issue by dude0001** [schireson/pytest-mock-resources#160]
I doubt this is a bug, and I'm just doing something wrong. I am not sure where else to comment for support. Sorry for the noise.

**Describe the bug**
Whenever I run a test that tries to use a fixture created with `create_redshift_fixture`, the test just hangs. When I debug, I see `create_redshift_fixture` is executed and returns and hangs shortly after that. I never see a container start up until I kill the test session, then I see the container running.

**Environment**
* Host OS: Ubuntu
* Docker image if applicable: postgres:9.6.10-alpine
* Python Version: 3.9.12
* Virtualenv/Pyenv etc.. if applicable: venv

**To Reproduce**

conftest.py

```
import pytest
from pytest_mock_resources import create_redshift_fixture

redshift_fixture = create_redshift_fixture()
```

test_mymoudle.py

```
def test_func(redshift_fixture):
    # I can duplicate trying to access the fixture or just an empty test as shown.
    assert True == True
```

**Expected behavior**
Test not to hang.

**Actual Behavior**
Test hangs

**Additional context**

**Comment by DanCardin** [comment]
until you kill the test session, huh! Do you have the same issue with `create_postgres_fixture`? Alternatively, what happens if you run `pmr redshift` ahead of time (this is meant to spin up a compatible, persistent container that's used across test runs)?

Finally, what version of the lib are you using? I'm assuming the most recent, which switched to a different underlying docker library, so there might be some new unseen issue.

Without the answers to those questions, my default assumption would be there's something different about your docker setup relative to what we use/test on.

I might have said you're failing to communicate with the container correctly and may need to configure `PYTEST_MOCK_RESOURCES_HOST`, but if the container isn't starting up until you kill the test run that sounds more like a bug or docker issues.

**Comment by dude0001** [comment]
I agree it is odd the container stops after I stop the tests. I am not using async. However, almost has the symptom of an async not being awaited somewhere blocking PMR and when I kill the task the logic to start the container is allowed to run. I did try [v2.2.7](https://github.com/schireson/pytest-mock-resources/releases/tag/v2.2.7) with the same result. I set `PYTEST_MOCK_RESOURCE_HOST` to a couple of different values but it did not have any effect for me.

I did try running `pmr redshift` before starting the tests. It takes a few minutes, but the container did start. When I then run the tests I see the same hanging though.  I looked at the logs on the container and I see some suspect messages like "database system is shut down". If the Postgress DB is down, that would connect to your theory of the tests not being able to reach the DB and stuck in some connect retry policy or something.

Are these logs normal or is there any way to troubleshoot this more? Is there any difference between using `pip install "pytest-mock-resources[postgres, redshift]"` and `pip install "pytest-mock-resources[postgres-binary, redshift]"`?

```
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... No usable system locales were found.
Use the option "--debug" to see details.
ok
sh: locale: not found

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
LOG:  database system was shut down at 2022-08-10 23:22:18 UTC
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

waiting for server to start....LOG:  database system was shut down at 2022-08-10 23:22:16 UTC
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down...LOG:  received fast shutdown request
.LOG:  aborting any active transactions
LOG:  autovacuum launcher shutting down
LOG:  shutting down
LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.
```

**Comment by dude0001** [comment]
Interestingly I walked away for a bit and `pmr redshift` had finally returned with an error. I am also now able to connect to Postgress inside the container with Data Grip. Running the tests again they still hang.

```
pmr redshift
Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3280, in _wrap_pool_connect
    return fn()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 310, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
    rec = pool._do_get()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 146, in _do_get
    self._dec_overflow()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
    return _ConnectionRecord(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
    self.__connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 590, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 597, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not connect to server: No route to host
        Is the server running on host "host.docker.internal" (10.0.0.251) and accepting
        TCP/IP connections on port 5532?


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/redshift.py", line 63, in check_fn
    get_sqlalchemy_engine(self, self.root_database)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/postgres.py", line 91, in get_sqlalchemy_engine
    engine.connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3234, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3313, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3283, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2117, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3280, in _wrap_pool_connect
    return fn()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 310, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
    rec = pool._do_get()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 146, in _do_get
    self._dec_overflow()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
    return _ConnectionRecord(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
    self.__connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 590, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 597, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not connect to server: No route to host
        Is the server running on host "host.docker.internal" (10.0.0.251) and accepting
        TCP/IP connections on port 5532?

(Background on this error at: https://sqlalche.me/e/14/e3q8)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/base.py", line 106, in wait_for_container
    retry(check_fn, retries=1, interval=interval, on_exc=ContainerCheckFailed)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/base.py", line 23, in retry
    result = func(*args, **kwargs)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/redshift.py", line 65, in check_fn
    raise ContainerCheckFailed(
pytest_mock_resources.container.base.ContainerCheckFailed: Unable to connect to a presumed Redshift test container via given config: RedshiftConfig(root_database='dev', username='user', ci_port=5432, host='host.docker.internal', password='password', image='postgres:9.6.10-alpine', port=5532)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3280, in _wrap_pool_connect
    return fn()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 310, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
    rec = pool._do_get()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 146, in _do_get
    self._dec_overflow()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
    return _ConnectionRecord(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
    self.__connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 590, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 597, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not connect to server: Connection timed out
        Is the server running on host "host.docker.internal" (10.0.0.251) and accepting
        TCP/IP connections on port 5532?


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/redshift.py", line 63, in check_fn
    get_sqlalchemy_engine(self, self.root_database)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/postgres.py", line 91, in get_sqlalchemy_engine
    engine.connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3234, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3313, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3283, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 2117, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/base.py", line 3280, in _wrap_pool_connect
    return fn()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 310, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
    rec = pool._do_get()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 146, in _do_get
    self._dec_overflow()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/impl.py", line 143, in _do_get
    return self._create_connection()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
    return _ConnectionRecord(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
    self.__connect()
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/util/compat.py", line 208, in raise_
    raise exception
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/create.py", line 590, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/sqlalchemy/engine/default.py", line 597, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not connect to server: Connection timed out
        Is the server running on host "host.docker.internal" (10.0.0.251) and accepting
        TCP/IP connections on port 5532?

(Background on this error at: https://sqlalche.me/e/14/e3q8)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mlambert/venvs/myapp/bin/pmr", line 8, in <module>
    sys.exit(main())
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/cli.py", line 65, in main
    execute(fixture_type, pytestconfig, start=start, stop=stop)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/cli.py", line 92, in execute
    for _ in generator:
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/base.py", line 76, in get_container
    container = wait_for_container(
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/base.py", line 123, in wait_for_container
    retry(check_fn, retries=retries, interval=interval, on_exc=ContainerCheckFailed)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/base.py", line 23, in retry
    result = func(*args, **kwargs)
  File "/home/mlambert/venvs/myapp/lib/python3.9/site-packages/pytest_mock_resources/container/redshift.py", line 65, in check_fn
    raise ContainerCheckFailed(
pytest_mock_resources.container.base.ContainerCheckFailed: Unable to connect to a presumed Redshift test container via given config: RedshiftConfig(root_database='dev', username='user', ci_port=5432, host='host.docker.internal', password='password', image='postgres:9.6.10-alpine', port=5532)
```

**Comment by dude0001** [comment]
Sorry for all the posts. I put a breakpoint here https://github.com/schireson/pytest-mock-resources/blob/35c794881823e67d8485269cba945a0bb1565813/src/pytest_mock_resources/fixture/database/relational/redshift/__init__.py#L26 and saw pmr_redshift_config.host is host.docker.internal similar to what the logs above say. If I change that to localhost and then continue, the tests run! I try to connect from Data Grip using host.docker.internal as the host, and I can no longer connect to Postgres in the container. I do have PYTEST_MOCK_RESOURCES_HOST=localhost in my environment variables, so I debugged some more to see where why it is using host.docker.internal as the host still.

It turns out the call `socket.gethostbyname(_DOCKER_HOST)` here (_DOCKER_HOST = "host.docker.internal") https://github.com/schireson/pytest-mock-resources/blob/35c794881823e67d8485269cba945a0bb1565813/src/pytest_mock_resources/config.py#L16  is returning a IP that isn't routable to inside the container. And if the code is able to resolve host.docker.internal to an IP here then the code here https://github.com/schireson/pytest-mock-resources/blob/35c794881823e67d8485269cba945a0bb1565813/src/pytest_mock_resources/config.py#L95 returns host.docker.internal instead of the value of environment variable PYTEST_MOCK_RESOURCES_HOST.

Is the fix to return the value of environment variable PYTEST_MOCK_RESOURCES_HOST here if it is set, no matter what. Otherwise, the environment variable is ineffective here. I am happy to put in a PR if this change sounds okay.

**Comment by DanCardin** [comment]
postgres vs postgres-binary is just the different psycopg2 options, fwiw. I do see the pr though, i'll take a look

**Comment by DanCardin** [comment]
Merged your PR, released in 2.4.4

**Comment by dude0001** [comment]
The PR and v2.4.4 resolves this issue for me. The container starts up within seconds and I can run tests with a workaround for my other issue #161. I'll close this and see what I can come up with for #161. Thank you for this project and your help, looks pretty cool so far.
